Mathlib/Data/Subtype.lean

```diff
@@ -29,6 +30,13 @@ namespace Subtype
 
 variable {α β γ : Sort*} {p q : α → Prop}
 
+#align subtype.eq Subtype.eq
+#align subtype.eta Subtype.eta
+
+#noalign subtype.tag_irrelevant
+#noalign subtype.exists_of_subtype
+#noalign subtype.inhabited
```

Comment:
I believe we don't like `#noalign`s, and prefer lemmas that roughly match, e.g. `Subtype.instInhabitedSubtype` (which probably matches exactly?)

Reply:
```suggestion
#align subtype.inhabited Subtype.instInhabitedSubtype
```
(how is that even an instance, though?)

Reply:
Fair point, I don't think it can ever be succesfully applied.